dev/app.js

```diff
@@ -122,6 +122,8 @@ search.addWidget(
 
 search.addWidget(
   instantsearch.widgets.refinementList({
+    collapsable: true,
+    collapsed: true,
```

Comment:
Oh that makes sense. I don't really like what we did for the URL sync but that could be consistent with what you want: 
 - type for collapsable `boolean|object`
 - having an object (even an empty one) is equivalent to having `true`
 - the object can contain an attribute `defaultState:boolean`